docs/sources/lids/template.md

```diff
@@ -0,0 +1,51 @@
+# Title
+
+> _NOTE: the file should be named `_<your-title>.md` and be placed in the `docs/sources/lids` directory. 
+Once accepted, it will be assigned a LID number and the file will be renamed by the sponsor.<br>
+> **Please remove this section before submitting, and retain the structure of this document.**_
+
+**Author:** Name (email)
+
+**Date:** MM/YYYY
+
+**Sponsor(s):** @username of maintainer(s) willing to shepherd this LID
+
+**Type:** Feature|Process
+
+**Status:** Draft|In Discussion|Accepted|Rejected|Completed
+
+**Related issues/PRs:**
+
+**Thread from [`#loki-dev` channel on Slack](https://slack.grafana.com):**
+
+---
+
+## Background
+
+_Describe the key facts and link to any documents, architecture diagrams or code. Keep this short, just a paragraph or two._
+
+## Problem Statement
+
+_Describe why the above is a problem we have now or will have in the future. Just a paragraph or two._
+
+## Goals
+
+_Describe the goals that we wish to achieve when considering the options available. Be clear about what is important and what could be left to the future. Non-goals are also useful to list here to clarify intent._
```

Comment:
I suggest to have a separate Non-Goals section here. From experience it is good to have an anchor when discussing an improvement on what we are **not** solving here. It served us well in the Loki operator enhancement proposal process so far.

Reply:
Agree, non-goals can be very helpful to scope a feature. Making the Non-Goals optional would make sense IMO

Reply:
Done :+1: